client/src/components/SupermarketDisplay.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+
+function SupermarketDisplay({ supermarket }) {
+  return (
+    <Container>
+      <div>
+        <h3>{supermarket.name}</h3>
+      </div>
+      <div>
+        <p>
+          Products found: {supermarket.matchingProductsFound} of{" "}
+          {supermarket.listProductCount}
+        </p>
+        <p>{supermarket.totalPrice.toFixed(2)}€</p>
+      </div>
+    </Container>
+  );
+}
+
+export default SupermarketDisplay;
+SupermarketDisplay.propTypes = {
+  supermarket: PropTypes.any,
```

Comment:
You can add the exact shape of a supermarket here:

```suggestion
  supermarket: PropTypes.shape({
      name: PropTypes.string,
      matchingProductsFound: PropTypes.number,
      listProductCount: PropTypes.number,
      totalPrice: PropTypes.number,
  }),
```

`listProductCount` sounds a little confusing and more like a function.

Reply:
very good to know!

Reply:
Nice. Thanks for the info here. :)

Reply:
vey nice. Thank you very much. Already changed it and will keep it mind for future naming